src/interfaces.ts

```diff
@@ -0,0 +1,52 @@
+import React, { Component } from 'react';
+
+import Tester from './tester';
+import ConfigurationClass from './ConfigurationClass';
+
+export type ComponentClass = React.FC | { new(): Component<any> };
+
+export interface IMountOps {
+  async?: boolean;
+}
+
+export interface IWrapper {
+  component: ComponentClass;
+  name: string;
+  props: object;
+  renderChildren?: boolean;
+}
+
+export interface IHook extends IWrapper {
+  [key: string]: any;
+  onBeforeMount: (tester: Tester, ops: IMountOps) => void;
+  onInit: (tester: Tester) => void;
+  props: object | (() => void); // fn() allows this.AppState to be set for e.g
+  shortCuts: { [shortCutName: string]: () => void };
```

Comment:
For both `shortCuts` and `wrapper`, I'm not even sure if these two are really returning what was documented.
If I'm right, sorry about that. Uhm... I haven't used Tester for 2 months so I'm so confused right now 😆 

I'd say we can treat this PR as a first pass that's gonna require fine tuning, so it's not a big deal as long as it doesn't break anything. If you want to go deeper into this, fine, but I don'T think it's worth it as the moment.

Reply:
Also you know what, I think they're not even used since I can't find anything related to that in the doc.
I'll create an issue for it, just to double-check if they're doing something and remove if they're useless.

no big deal for now though.